cloud/amazon/ec2_eni.py

```diff
@@ -341,6 +355,37 @@ def get_sec_group_list(groups):
 
     return remote_security_groups
 
+def make_securitygroup_list(connection, module, group_list,subnet_id,eni_id):
```

Comment:
I can see the value of having a shared function... both this, and tagging would be better centrally.

That said, implementing this as shared code has a couple of gotchas:
1. The significant differences in implementation of securitygroup handling between modules today, and handling all of those use cases.
2. The lack of somewhere for that to go except module_utils/ec2.py which currently handles only connection.

I'd be happy to contribute something, but see that as a much bigger piece of work, and it taking longer to get done.

(also, ec2_eni with these mods can be used in an ansible 1.9 environment - that's useful to some people)